### ISSUE TITLE
Can't find it in Firefox Addons Search

### ISSUE THREAD

**Issue by fredybp** [Hau-Hau/firefox-tab-suspender#107]
Hi, I was looking to get the extension on my other computer and could not find it...

I saw that The Great Suspender had some security issues and was removed from Chrome store, did this happen to this Addon as well?

I love the Tab Suspender, thank you to everyone in this project :D

**Comment by Jonta** [comment]
iirc here: https://addons.mozilla.org/en-US/firefox/addon/ff-tab-suspender/

I think they removed it in anticipation of their upcoming better extension

Please restore this to the addons-page

I've tried the current alternatives, and they're all pretty much unusable for my needs

**Comment by fredybp** [comment]
I'm getting `Oops! We can’t find that page` on that page

**Comment by Jonta** [comment]
> I'm getting `Oops! We can’t find that page` on that page

Exactly.

Rehosted here: https://github.com/Jonta/rehost-of-tabsuspender-by-hau-hau

**Comment by Hau-Hau** [comment]
Hello, .xpi is also available here: https://github.com/Hau-Hau/firefox-tab-suspender/releases/tag/3.7.4

**Comment by ProgrammingLife** [comment]
@Hau-Hau Are gonna make new improvements to this addon soon? Or is it abandoned addon?

**Comment by Hau-Hau** [comment]
@ProgrammingLife 
No, project isn't abandoned. I'm still working on new version. Because of work and university I don't have much free time but fortunately I will graduate soon.
